src/main/resources/xjc/medline/medline.xsd

```diff
@@ -1,314 +1,488 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- Source: https://github.com/lstern/psilibrary/blob/master/doc/medline/medline.xsd-->
-<!--
-     This is the Current DTD which NLM has written for 
-  External  Use.  If you are a NCBI User, use the information
-  from the PubmedArticleSet.
-  
-  Comments and suggestions are welcome.
-  (May 9, 2000)
-  
-  Corrections:
-  ~~~~~~~~~~~
-  Oct. 09 2002 
-  - "PubMedArticle" has been renamed to "PubmedArticle"
-  - All referencies to "PubMedArticle" has been removed
-  - "ProviderId" has been removed from PubmedData
-  - "URL" has been removed from PubmdeData
-  
-  	$Id: pubmed_160101.dtd 480858 2015-10-05 15:12:19Z korobtch $
-  
--->
-<!-- ================================================================= -->
-<!-- ================================================================= -->
-<!-- Reference to Where the MEDLINECITATION DTD is located -->
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
-  <xs:include schemaLocation="nlmmedlinecitationset_160101.xsd"/>
-  <xs:include schemaLocation="bookdoc_160101.xsd"/>
-  <!-- ================================================================= -->
-  <xs:simpleType name="iso.language.codes">
-    <xs:restriction base="xs:token">
-      <xs:enumeration value="AF"/>
```

Comment:
Why is the `medline.xsd` was regenerated? These codes do not exist in the new file. -- Think, there has to be another way to achieve the goal.

Reply:
I commited an old version. I managed to recreate the xsd now from the dtd and a bunch of [other required dtds](https://github.com/biopython/biopython/tree/master/Bio/Entrez/DTDs) and adjusting the headers. However, the orginal error  regarding the mathml remains.

We already use our own xjc plugin which does also execute the ant task.